src/mlmd/generated/ml_metadata/proto/metadata_store_pb.js

```diff
@@ -7,9865 +7,10 @@
  * @public
  */
 // GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
 
 var jspb = require('google-protobuf');
 var goog = jspb;
 var global = Function('return this')();
 
-goog.exportSymbol('proto.ml_metadata.AnyArtifactStructType', null, global);
```

Comment:
It seems a little weird, this file is now almost empty.

Reply:
Thanks for pointing it out. There was an async bug on previous script which will lead to generate protobuf before file got downloaded. I split the scripts into two parts to have different functions: 
1. replace the proto files with target version(newly added one).
2. generate protobufs(same as before).

Now the protobuf regen steps is updated on README.md with two steps. PTAL!